### ISSUE TITLE
Swords dealing no damage?

### ISSUE THREAD

**Issue by noah12333** [dmulloy2/ProtocolLib#285]
_Follow this template except for feature requests. Use pastebin when providing /protocol dump and any relevant errors._

Make sure you've done the following:
- [x] You're using the latest build for your server version
- [x] This isn't an issue caused by another plugin
- [x] You've checked for duplicate issues

Debug paste link: NA

Description and relevant errors: I have no idea why, but ProtocolLib was causing the swords in Minecraft (all of them) to deal only half a heart of damage on any type of armor (and without) It seems like the most random to happen from this plugin, but I did confirm that it was from ProtocolLib

**Comment by MiniDigger** [comment]
I don't think that protocol lib is doing it but a plugin that uses protocol lib. 
can you try to reproduce with only protocol lib?

**Comment by noah12333** [comment]
Yes I did, I was using it for Spectator and I even tried it without that plugin. The only other plugin that was in the folder with ProtocolLib was FactionsUUID

**Comment by dmulloy2** [comment]
I need some more information. Run ````/protocol dump```` and paste the output using [pastebin](http://pastebin.com/).
